packages/nodes-base/nodes/GraphQL/GraphQL.node.ts

```diff
@@ -159,6 +197,214 @@ export class GraphQL implements INodeType {
 				placeholder: 'http://example.com/graphql',
 				description: 'The GraphQL endpoint',
 				required: true,
+				displayOptions: {
+					show: {
+						connectionMode: ['http'],
+					},
+				},
+			},
+			{
+				displayName: 'WebSocket URL',
+				name: 'websocketUrl',
+				type: 'string',
+				default: '',
+				placeholder: 'wss://example.com/graphql',
+				description: 'The WebSocket URL for GraphQL subscriptions',
+				required: true,
+				displayOptions: {
+					show: {
+						connectionMode: ['websocket'],
+					},
+				},
+			},
+			{
+				displayName: 'Ignore SSL Issues (Insecure)',
+				name: 'websocketAllowUnauthorizedCerts',
+				type: 'boolean',
+				default: false,
+				description: 'Whether to connect even if SSL certificate validation is not possible',
+				displayOptions: {
+					show: {
+						connectionMode: ['websocket'],
+					},
+				},
+			},
+			{
+				displayName: 'Request Format',
+				name: 'websocketRequestFormat',
+				type: 'options',
+				required: true,
+				options: [
+					{
+						name: 'JSON (Recommended)',
+						value: 'json',
+						description: 'JSON object with query, variables, and operationName properties',
+					},
+					{
+						name: 'GraphQL (Raw)',
+						value: 'graphql',
+						description: 'Raw GraphQL query string',
+					},
+				],
+				default: 'json',
+				description: 'The request format for the subscription payload',
+				displayOptions: {
+					show: {
+						connectionMode: ['websocket'],
+					},
+				},
+			},
+			{
+				displayName: 'Query',
+				name: 'websocketQuery',
```

Comment:
Same as the others, This also applies to most of the options below that can be shared.